LayoutKit/LayoutArrangement.swift

```diff
@@ -151,8 +151,9 @@ extension LayoutArrangement: CustomDebugStringConvertible {
         let t = String(repeatElement(" ", count: indent * 2))
         let sublayouts = self.sublayouts.isEmpty
             ? ""
-            : self.sublayouts.map { $0._debugDescription(indent + 1) }.joined(separator: "")
-        return"\(t)LayoutArrangement: \(frame)\n\(sublayouts)"
+            : self.sublayouts.map { $0._debugDescription(indent + 1) }.joined()
+        let layoutName = String(describing: layout).components(separatedBy: ".").last!
```

Comment:
I know this is just debug code and there probably should always be a period in the name, it would still be a tiny bit safer to do `fullName.components(separatedBy: ".").last ?? fullName`.

One more thing, while playing with your test playground file I thought it might also help to add the viewReuseId into this output. Up to you though whether this would be adding too much text into the output thereby hurting its readability.